packages/grafana-data/src/types/featureToggles.gen.ts

```diff
@@ -43,6 +43,7 @@ export interface FeatureToggles {
   lokiLive?: boolean;
   swaggerUi?: boolean;
   featureHighlights?: boolean;
+  saveWithDiff?: boolean;
```

Comment:
This is here so we can test both flavors for a bit -- but I think we should remove modals